static/src/javascripts/projects/common/modules/identity/consent-journey.js

```diff
@@ -58,11 +73,28 @@ const showJourneyAlert = (journeyEl: HTMLElement): void => {
         formEl.addEventListener('submit', ev => {
             if (ev.isTrusted) {
                 ev.preventDefault();
-                Promise.all([
-                    shouldDisplayNewsletterWarning(journeyEl),
-                    shouldDisplayConsentWarning(journeyEl),
-                ]).then(([emptyNewsletters, emptyMarketing]) => {
-                    if (emptyNewsletters && emptyMarketing) {
+                const uncheckBoxesPromise = Promise.all([
```

Comment:
not sure if you should declare this at all instead of just letting it chain off

Reply:
Yeah chain off is better